blueprints/tags.md

```diff
@@ -0,0 +1,421 @@
+	}
+}
+```
+
+###### DELETE
+Deletes a Tag.
+
+- *Required Roles* Operations or Admin
+- *Response Type* Object
+
+**Request Structure**
+The single route parameter `Tag Name` must be the name of an existing
+Tag.
+
+This method of this endpoint provides no query string parameters.
+
+*Request Example*
+```http
+DELETE /api/4.0/tags/Bar HTTP/1.1
+Host: trafficops.infra.ciab.test
+Accept: application/json, */*;q=0.9
+Cookie: mojolicious=...
+
+```
+
+**Response Structure**
+The response is a representation of the deleted Tag.
+
+*Response Example*
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+Server: Traffic Ops/5.0
+Date: Tue, 23 Jun 2020 20:46:57 GMT
+Transfer-Encoding: chunked
+
+{ "alerts": [{
+	"level": "success",
+	"text": "Deleted Tag 'Bar'"
+}],
+"response":
+	{
+		"name": "Bar"
+	}
+}
+```
+
+
+#### Client Impact
+The return data structures will change, but no code change to any Traffic Ops
+client should be necessary for the addition of Tags to existing objects.
+However, new methods/functions/procedures must be added to manipulate Tags
+themselves.
+
+### Traffic Portal Impact
+A new UI section, complete with tables and forms for editing and creation, will
+need to be added to accommodate the addition, editing and deletion of Tags.
+Also, existing tables and forms for now-taggable objects will need to be updated
+to accommodate the use of Tags. Each separate change can be done atomically, and
+since the new collections need not be required, they can be done in non-invasive
+fashion of arbitrary order (provided Tag manipulation itself is done first)
+without disrupting any existing processes or data.
+
+## Documentation Impact
+Tags will need to be documented in the data model, both themselves and on the
+modeled objects which now contain them. Also, API endpoints will need to updated
+to reflect the new return structures, where applicable.
+
+## Testing Impact
+All new Traffic Ops client methods/functions/procedures will need to be
+accompanied by corresponding tests, and ideally some part of the route handlers
+will also be testable by unit tests.
+
+## Performance Impact
+For most objects this would merely add a column to the database query structure,
+so the performance impact is expected to be negligible.
+
+## Security Impact
+Tags have no implied functionality, and expose no sensitive data, so there is
+expected to be no security impact.
+
+## Upgrade Impact
+This will require probably multiple database migrations (or possibly one if the
+implementer is feeling brave), but should require no change to upgrade processes
+nor cause any problems on roll-back, as the only lost data would be the new Tags
+that would be inaccessible in an older version anyway.
+
+## Operations Impact
+Tags have no operational meaning, but operators may need to be made aware of
+Tags' existences and different organizations may place their own, proprietary
+importance on a Tag or Tags that they may then need to communicate to operators.
```

Comment:
Correct, tags have no internal TO logic associated and as such are only meaningful to systems outside TO which apply logic based on tags.  Someday if the corresponding concept of `Policies` exists, then that's a different story.